master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/ha/MetaHandler.java

```diff
@@ -118,6 +118,16 @@ public ResourceResponse handleWriteRequest(ResourceProtos.ResourceRequest reques
           metaSystem.updateUnregisterShuffleMeta(shuffleKey);
           break;
 
+        case BatchUnRegisterShuffle:
+          List<String> shuffleKeys =
+              request.getBatchUnregisterShuffleRequest().getShuffleKeysList();
+          shuffleKeys.forEach(
+              key -> {
+                LOG.debug("Handle unregister shuffle for {}", key);
```

Comment:
pull this out of the foreach and dump all in one shot ? debug logging is already noisy :-)

Reply:
+1.

Reply:
ok thanks！